pkgs/tools/text/mdbook-i18n-helpers/default.nix

```diff
@@ -5,17 +5,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "mdbook-i18n-helpers";
-  version = "0.3.4";
+  version = "0.3.5";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "mdbook-i18n-helpers";
-    # TODO fix once upstream uses semver for tags again
-    rev = "refs/tags/mdbook-i18n-helpers-${version}";
-    hash = "sha256-nIavOBWPyCmCLwPcDbjjG1sa71HObGnhms9sSCAR2iA=";
+    # Upstream forgot to bump the version in Cargo.lock in 0.3.5
+    # TODO use git tag again on next release
+    # https://github.com/google/mdbook-i18n-helpers/pull/216#issuecomment-2255799969
+    rev = "d80451f48af3f495ad78512865cbe5943928b96c";
+    hash = "sha256-JMVxSNhMLMfsP+cJkuRUWknZZM7Ji0RjbLqrgjHeg6g=";
```

Comment:
Nice that stuff like this also happens to google engineers :laughing:

Reply:
Yeah, we're all human :smile: